docs/source/references/python.rst

```diff
@@ -85,6 +85,11 @@ Python API Reference
 
   **General methods**
 
+  .. function:: setOpenVINOVersion()
+
+    Sets explicit openvino version. Example: `pipeline = setOpenVINOVersion(version = dai.OpenVINO.Version.VERSION_2021_2)`
```

Comment:
Type: `pipeline.setOpenVINOVersion(version = dai.OpenVINO.Version.VERSION_2021_2)`
Although this will be documented automatically so doesn't matter as much